### PR TITLE
add two missing publications

### PR DESCRIPTION
**Pull request by tamiko**

### CODE REVIEW

File: publications-2023.bib [hunk 1]
```diff
@@ -508,6 +521,19 @@ @Article{2023:luo:fast
...
+@Article{2023:maier.shadid.tomas:euler-poisson,
+  author  = {Matthias Maier and John Shadid and Ignacio Tomas},
+  title   = {Structure-preserving finite-element schemes for the {E}uler-{P}oisson equations},
+  journal = {Communications in Computational Physics},
+  year    = 2023,
+  volume  = 33,
+  pages   = {647--691},
+  month   = apr,
+  doi     = {10.4208/cicp.OA-2022-0205},
+  url     = {10.4208/cicp.OA-2022-0205},
```
Comment (marcfehling) [inline publications-2023.bib:533]:
```suggestion
  url     = {https://global-sci.org/intro/article_detail/cicp/21656.html},
```

**Review by marcfehling — APPROVED** [review]
Looks good. Please adjust the URL

File: publications-2023.bib [hunk 2]
```diff
@@ -508,6 +521,19 @@ @Article{2023:luo:fast
   publisher = {Elsevier {BV}}
 }
 
+@Article{2023:maier.shadid.tomas:euler-poisson,
+  author  = {Matthias Maier and John Shadid and Ignacio Tomas},
```
Comment (bangerth) [inline publications-2023.bib:525]: Oh, you wrote a paper with John Shadid! I like John a lot!

Comment (tamiko) [reply publications-2023.bib:525]: :smiley:

**Pull request merged**